### ISSUE TITLE
Troubleshooting follow_me.py

### ISSUE THREAD

**Issue by esmalls** [dronekit/dronekit-python#10]
Hi Kevin,

Our team was able to run the follow_me.py example without debugging errors in two different ways. The first way we ran it, was exactly as the follow_me.py was written, and the second way was hardcoding one GPS coordinate in the Location function. However, in both cases we received the proper debugging output but the motors did not move. 

The first way we ran it, was exactly as the follow_me.py was written. Our GPS USB dongle was providing GPS coordinates every two seconds, those coordinates were displayed, and the output showed “Got MAVLink msg: MISSION_ACK... etc.

The second way we ran it was with all the gpsd functions commented out and by hard-coding one GPS coordinate in the Location function.

After educating ourselves on your API, we believe that we have narrowed down the problem to being the goto function does not end up forcing the drone to move the motors.

Do you have any thoughts on why this could be happening.

We speculate that the problem may be that the drone is going into Guided mode. We believe that this is your intent. However, from our understanding, the drone will only go to locations autonomously in AUTO mode and only after the waypoints are saved and an rc command starts the motors. 

Again, we do not fully understand what is wrong and why the command itself seems to be received by the APM, but the motors are not moving.

Again, we are able to get small_demo.py to move the motors properly, so we know it is not a problem with our wiring or our connection between the APM and our Raspberry Pi.

Any help is appreciated,

Thanks!

**Comment by mrpollo** [comment]
If this isn't yet resolved lets create a new issue to troubleshoot

**Comment by tcr3dr** [comment]
Since this may not have gotten rolled back into any documentation, this is what I'm using for some SITL tests:

```
param set FS_THR_ENABLE 0
param set FS_GCS_ENABLE 0
param set ARMING_CHECK 0
```

Which seemed to be successful. I can imagine scenarios where a DISARMING MOTORS trigger would still occur, but have nothing that causes that effect so far.

**Comment by hamishwillee** [comment]
@tcr3dr Setting ARMING_CHECK 0 is a big no no for real code. Might be worth adding a note in the test code to make it clear that this isn't something to do normally (since some people might copy it).

**Comment by tcr3dr** [comment]
@hamishwillee Can confirm. Basically a big warning around the ARMING_CHECK section on SITL will help. http://android.dronekit.io/getting_started.html

**Comment by hamishwillee** [comment]
@tcr3dr Thanks. I will add note for Android (https://github.com/DroidPlanner/DroneKit-Android/issues/199). There is already a note for Python but it could be more prominent (https://github.com/diydrones/dronekit-python/issues/205).

**Comment by tcr3dr** [comment]
Going to close this since it's API references are fairly outdated.

**Comment by hamishwillee** [comment]
Good call.
